src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -71,13 +87,54 @@ public void close() throws Exception {
     // Mostly for JUnit tests
     motor.close();
     motor2.close();
-    
+  }
+
+  public double getConeDist() {
+    return timeOfFlightSensor.getRawDistance0().classDistance;
+  }
+
+  public double getCubeDist() {
+    return timeOfFlightSensor.getRawDistance1().classDistance;
+  }
+
+  public void changeFlightState() {
+    double coneDistance = getConeDist();
+    double cubeDistance = getCubeDist();
+
+    boolean coneSensorUp = coneDistance != -1;
+    boolean cubeSensorUp = cubeDistance != -1;
+
+    SmartDashboard.putNumber("Cone Distance", coneDistance);
```

Comment:
There is an incoming class in release 3 that is debuglog.  Should look at that to reduce shuffleboard changes we need before competition